pyakeneo/result.py

```diff
@@ -44,16 +58,16 @@ def get_page_items(self):
 
     def fetch_next_page(self):
         """Return True if a next page exists. Returns False otherwise."""
-        if self._link_next:
+        if self.is_paginated(self._items) and self._link_next:
             r = self._session.get(self._link_next)
-            if r.status_code == 200:
-                (
-                    self._link_first,
-                    self._link_self,
-                    self._link_next,
-                    self._items,
-                    self._count,
-                ) = Result.parse_page(json.loads(r.text))
+            if r.status_code == requests.codes.ok:
```

Comment:
`requests` has a function for it

```suggestion
            if is_success(r.status_code):
```

Reply:
I didn't find the method you suggest, instead I used the ["ok" property](https://requests.readthedocs.io/en/latest/api/#requests.Response.ok)

Reply:
oh, sorry, it's from DRF :sweat_smile: